cloud/pkg/cloudhub/servers/udsserver/server.go

```diff
@@ -42,7 +42,10 @@ func StartServer(address string) {
 	})
 
 	klog.Info("start unix domain socket server")
-	uds.StartServer()
+	if err := uds.StartServer(); err != nil {
+		klog.Errorf("failed to start uds server: %v", err)
```

Comment:
I am not sure about this, maybe fatal? 

cc @fisherxu  @kevin-wangzefeng  what do you think?